src/engine/ruleset/decoders/nginx/nginx-access.yml

```diff
@@ -0,0 +1,174 @@
+---
+name: decoder/nginx-access/0
+
+metadata:
+  title: NGINX Access Decoder
+  description: Parses nginx access logs
+  references: "https://docs.nginx.com/nginx/admin-guide/monitoring/logging/#access_log"
+  product.name: nginx
+  product.versions:
+    - nginx-1.23.2  #latest version when developing
+  author:
+      name: Wazuh, Inc.
+      url: https://wazuh.com
+      date: 2023/01/23
+
+sources:
+  - decoder/nginx-access-base/0
+
+check:
+  - ~destination_address: +ef_exists
+
+parse:
```

Comment:
Can there be more than 3 ips?

Reply:
It can, but the only example I could find was the one with 4, that's why the second expression was defined.
Besides that, right now we cannot use a variable length mapping, we should know how many IPs or domains we could handle before parsing it.